nixos/tests/restic.nix

```diff
@@ -98,6 +101,8 @@ import ./make-test-python.nix (
           '${pkgs.restic}/bin/restic -r ${rcloneRepository} -p ${passwordFile} snapshots -c | grep -e "^1 snapshot"',
           "systemctl start restic-backups-custompackage.service",
           "grep 'backup .* /opt' /tmp/fake-restic.log",
+          "cat /tmp/fake-restic.log",
```

Comment:
Was this used for debugging? I don't think we need it in test.

Reply:
Agreed. This seems not used in test.

Reply:
Ah, yes, you're right. Fixed!